common/envs/prod.ts

```diff
@@ -37,6 +40,11 @@ export const PROD_CONFIG: EnvConfig = {
     appId: '1:128925704902:web:f61f86944d8ffa2a642dc7',
     measurementId: 'G-SSFK1Q138D',
   },
+  // TODO: fill in real endpoints for prod
```

Comment:
I guess you would get the new endpoints for prod after deploying those v2 functions, but not before?

Probably the suffix of the domain name is common across our prod app so the endpoint going forward could be predicted by switching out the function name.

Reply:
I suspect that's correct, but it's not documented anywhere I can see.